build_library/vm_image_util.sh

```diff
@@ -278,6 +280,13 @@ IMG_azure_OEM_USE=azure
 IMG_azure_OEM_PACKAGE=common-oem-files
 IMG_azure_OEM_SYSEXT=oem-azure
 
+## hetzner
+IMG_hetzner_DISK_FORMAT=raw
```

Comment:
```suggestion
```
Since this is the default I think we can omit it

Reply:
Removed in [c0d1796](https://github.com/flatcar/scripts/pull/1880/commits/c0d179620f1f0e6c51fbd439486064b52c82f982)